File: UMA.App.IdentityManager/Identity/Queries/UpdateIdentityQuery.cs
using MediatR;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using UMA.App.IdentityManager.Identity.Models;

namespace UMA.App.IdentityManager.Identity.Queries
{
    public class UpdateIdentityQuery : IRequest<UpdateIdentityResult>
    {
        public Guid Id { get; set; }
        public string Login { get; set; }
        public string Password { get; set; }
    }
}

File: UMA.Web.Angular/UMA-Web-Angular/src/app/_entities/user/query/updateIdentity.query.ts
import { IRequest } from 'src/app/_models/interfaces/IRequest';
import { UpdateIdentityViewModel } from '../response/updateidentity.viewmodel';

export class UpdateIdentityQuery implements IRequest<UpdateIdentityViewModel> {
  _response?: UpdateIdentityViewModel;
  id: string;
  login: string;
  password: string;
}
